npm/src/upload/uploadform/upload-form.ts

```diff
@@ -399,10 +277,45 @@ export class UploadForm {
 
   private checkIfDroppedItemIsFolder(webkitEntry: any) {
     if (webkitEntry!.isFile) {
-      this.rejectUserItemSelection(
+      this.removeFilesAndDropShadow()
+      rejectUserItemSelection(
         this.warningMessages?.incorrectItemSelectedMessage,
+        this.warningMessages,
+        this.successMessage,
         "Only folders are allowed to be selected"
       )
     }
   }
+
+  private removeFilesAndDropShadow() {
```

Comment:
What does the "DropShadow" part of the name refer to? Looks like the function removes the selected files removes the drag over? Maybe rename this?